int64 getFromSlot
---
int64 currentTool